Cloudflare/gateway_http/ingest/parser.yml

```diff
@@ -32,10 +32,17 @@ stages:
       - set:
           event.type: [ "denied" ]
         filter: '{{json_event.message.Action == "deny"}}'
+      - set:
+          file.hash.sha256: '{{json_event.message.BlockedFileHash}}'
```

Comment:
Are you sure the hash is always a sha256 ?

Reply:
No... It's not documented, as far as I can tell.
In the tests I did, I got either an empty string or a sha256, and I assumed that it would always be.

Is there a way to handle this more robustly? A length check on the hash?